site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,196 @@
+pressed the nuclear button and released Project Prometheus, making yuzu use up to 6 to 7 CPU threads in ideal conditions 
+compared to the previous 2 to 3. Expect a performance boost in a lot of games, while others will stay mostly the same due to 
+being coded to only use a single thread of the possible 3 the Switch offers to games.
+[Pull Request #3955.](https://github.com/yuzu-emu/yuzu/pull/3955)
+
+Now, some clarifications are needed on this change. Multicore support can’t hit mainline for now due to incompatibilities with 
+its code and the Master branch of yuzu. Those are being resolved, but please have patience.
+Also, users with 2 cores and either 2 or 4 threads should not enable multicore as it will most likely result in a performance 
+loss for them due to the lack of physical cores on their CPUs. Our [hardware recommendations](https://yuzu-emu.org/help/quickstart/#hardware) 
+have been updated accordingly.
+
+## Unreal Engine fixes
+
+[Rodrigo](https://github.com/ReinUsesLisp) implemented rendering more than one slice of 3D textures, fixing the most glaring 
+issue in Unreal Engine 4 games, known as “the Rainbow”. This also improves the previous implementation that was used in 
+Xenoblade games.
+[Pull Request #4027.](https://github.com/yuzu-emu/yuzu/pull/4027)
+
+{{< single-title-imgs
+    "Your Excellency (OCTOPATH TRAVELER)"
+    "./01.png"
+    "./02.png"
+  >}}
+
+## Animal Crossing: New Horizons changes
+
+[Rodrigo](https://github.com/ReinUsesLisp) fixed Animal Crossing: New Horizons terrain borders in Vulkan by implementing 
+constant attributes. This is not a native extension, constant attributes has to be emulated in Vulkan as there is no 
+current official support for it.
+[Pull Request #3930.]( https://github.com/yuzu-emu/yuzu/pull/3930)
+
+{{< single-title-imgs
+    "Beautiful beaches, now in Vulkan too (Animal Crossing: New Horizons)"
+    "./04.png"
+    "./05.png"
+  >}}
+
+Some more “internal” fixes were also done to Nook Inc.’s escape package, [bunnei](https://github.com/bunnei) added a time zone 
+mechanism to show the correct time in game depending on where our users live, previously yuzu always assumed a GMT+0 timezone 
+was in place.
+[Pull Request #3909.](https://github.com/yuzu-emu/yuzu/pull/3909)
+
+[bunnei](https://github.com/bunnei) also improved the saving mechanism. Normal games save their data on each user profile, but 
+`Animal Crossing: New Horizons` does it in a “device” profile, so yuzu had to be accommodated for that.
+[Pull Request #3665.](https://github.com/yuzu-emu/yuzu/pull/3665)
+
+## Xenoblade specific fixes
+
+Rendering bugs are abundant in Xenoblade games, and they are not of the “easy to solve” sort due to how these games are coded. 
+[Rodrigo](https://github.com/ReinUsesLisp) managed to fix most of them by improving the texture cache, this is thanks to work done by [gdkchan](https://github.com/gdkchan) from [Ryujinx](https://github.com/Ryujinx/Ryujinx). You can see the results 
```

Comment:
Change `improving the texture cache` to `[improving the texture cache](https://github.com/yuzu-emu/yuzu/pull/3991)`